src/zoom.js

```diff
@@ -76,7 +77,6 @@ export default function() {
         .on("touchstart.zoom", touchstarted)
         .on("touchmove.zoom", touchmoved)
         .on("touchend.zoom touchcancel.zoom", touchended)
-        .style("touch-action", "none")
```

Comment:
Have not found the reason for this class. It unfortunately prevents these changes to apply. We can probably programatically add and remove it if it is necessary here.